website/static/css/style.css

```diff
@@ -718,23 +718,41 @@ div.ball-scale-blue>div {
     margin-bottom: 0;
 }
 
-/* To be added to osf-style: project overview -> componentQuickActions menu */
-#componentQuickActions ul.dropdown-menu > li > a:hover,
-#componentQuickActions ul.dropdown-menu > li > a:focus {
+/* To be added to osf-style: generic dropdown-menu */
+.generic-dropdown ul.dropdown-menu > li > a:hover,
+.generic-dropdown ul.dropdown-menu > li > a:focus {
     background-image: none;
-    background-color: #D9D9D9;
+    background-color: #D9D9D9 !important;
```

Comment:
As a general rule, if you have to add a `!important` to a CSS class, something is probably wrong. I'm not going to bump it out because of this, because CSS… ugh. But I wouldn't complain if you spend some more time evaluating the CSS chain and seeing if you can do this in a way that doesn't require the `!important`.